data.json

```diff
@@ -52,13 +61,17 @@
   {
     "name": "new_york",
     "url": "http://gis.ny.gov/gisdata/data/ds_921/SAM_Master_Statewide.gdb.zip",
-    "hash": "9a52775995af73fe5e6cd1aeaa9b52fbe9c35389c8657dbd75f99149caa10fd9",
+    "hash": "dda07a26bb25b5c900d0d1464ade072e3aa769372bf4d9bca75fa05ddae96122",
     "file": "SAM_Master_Statewide_Database.gdb",
     "count": 4761260,
     "fields": {
-      "Address": {
-        "type": "static",
-        "value": "AddressLabel"
+      "Number": {
+        "type": "multi",
+        "value": ["PrefixAddressNumber", "AddressNumber", "SuffixAddressNumber"]
+      },
+      "Street": {
+        "type": "multi",
+        "value": ["PreModifier", "PreDirectional", "PreType", "StreetName", "PostType", "PostDirectional", "PostModifier"]
```

Comment:
In the json object that gets produced by ogr2ogr on their raw data, the pre modifier comes first.

This is reflected in the following row address:
````
{
  "type": "Feature",
  "properties": {
    "NYSAddressPointID": "WYOM000548",
    "CountyID": "6180",
    "NYSStreetID": 437252580,
    "PrefixAddressNumber": null,
    "AddressNumber": 6860,
    "SuffixAddressNumber": null,
    "PreModifier": "Old",
    "PreDirectional": "E",
    "PreType": null,
    "SeparatorElement": null,
    "StreetName": "Arcade",
    "PostType": "Rd",
    "PostDirectional": null,
    "PostModifier": null,
    "SubAddress": null,
    "LandmarkName": null,
    "Building": null,
    "Floor": null,
    "Unit": null,
    "Room": null,
    "Seat": null,
    "Location": null,
    "ZipName": "Arcade",
    "State": "NY",
    "ZipCode": "14009",
    "PointType": "1",
    "AddressSource": "County AP",
    "SourceOfData": null,
    "DateUpdated": "2014/01/29 00:00:00",
    "PrimaryPoint": "Y",
    "CityTownName": "Arcade",
    "PlaceName": null,
    "PlaceType": null,
    "CompleteStreetName": "Old East Arcade Road",
    "AP_Flag": null,
    "Status": "Active",
    "AddressLabel": "6860 Old E Arcade Rd \r\n",
    "CR_UserName": null,
    "CR_DateTime": null,
    "AT_UserName": null,
    "AT_DateTime": null,
    "SP_UserName": null,
    "SP_DateTime": null,
    "CountyName": "Wyoming",
    "CLDXF_PreDirectional": "East",
    "CLDXF_PostType": "Road",
    "CLDXF_PostDirectional": null,
    "Latitude": 42.57014195419415,
    "Longitude": -78.34871794629433
  },
  "geometry": {
    "type": "Point",
    "coordinates": [
      225143.04760000017,
      4718518.536
    ]
  }
}
````

Where premodifier is ````Old```` and predirectional is ````E```` leading to ````6860 Old E Arcade Rd```` which intuitively sounds right to me.

Thoughts?